flexmeasures/ui/templates/base.html

```diff
@@ -1032,7 +1032,123 @@
                 toast.remove();
                 maybeHideCloseToastBtn();
             });            
-          }
+        }
+
+        // Fetch Account Details
+        async function getAccount(accountId) {
```

Comment:
We can use base.html to store a lot of JavaScript functions. It works.

We can also place them in ui/static/js/flexmeasures.js or better yet - to a more dedicated module in there, e.g. flex-context-and-model-dialogues.js (lines after 1091) and the three getters here could go on data-utils.js

Advantages:
- shorter base.html (easier reading)
- not all pages load these hundreds of lines
- maybe browsers can compress clean JavaScript files better